### PR TITLE
Commit of augmentation files for word stemmer

### PR DESCRIPTION
**Pull request by davidwanguoft**
Title.

Adding files for a word stemmer augmentation. See README for more details.

### CODE REVIEW

**Comment by mille-s** [comment]
Hi, thanks for the submission! Could you please describe a scenario in which we could use the stemming to better test models? I'm not sure how the created data could be used for training or testing purposes in the context of NL-Augmenter.

**Comment by kaustubhdhole** [comment]
I agree with @mille-s though I like the idea of using stemmed words. It might be useful for your case to point out some related work or motivation about stemming eg. is the meaning fully preserved at least in the eyes of the human? Here are a few comments:

File: transformations/word_stemmer/transformation.py [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    
+    Attributes:
+
+    """
+    tasks = [
+        TaskType.TEXT_CLASSIFICATION,
+        TaskType.TEXT_TO_TEXT_GENERATION,
+    ]
+    languages = ["en"]
+    heavy = True
```
Comment (kaustubhdhole) [inline transformations/word_stemmer/transformation.py:42]: Do you need heavy to be true here? Also, please add the keywords

File: transformations/word_stemmer/test.json [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+          "sentence": "hi eye were danc with humor."
+        }]
+      },
+      {
+        "class": "SimpleWordStemmer",
+        "inputs": {
+          "sentence": "She shook her head and danced away"
+        },
+        "outputs": [{
+          "sentence": "she shook her head and danc away"
```
Comment (kaustubhdhole) [inline transformations/word_stemmer/test.json:28]: If you could maintain the casing in the output, that might help make the transformation specific?

**Comment by kaustubhdhole** [comment]
Thanks @davidwanguoft, this transformation looks good for NL Augmenter. Please address the above comments and I am happy to merge. If possible, please perform the robustness evaluation too.

**Comment by kaustubhdhole** [comment]
@davidwanguoft ping!

**Pull request closed without merging**